api-server/common/models/User-Identity.js

```diff
@@ -41,10 +41,9 @@ export default function(UserIdent) {
       include: 'user'
     };
     // get the email from the auth0 (its expected from social providers)
-    const email =
-      profile && profile.emails && profile.emails[0]
-        ? profile.emails[0].value
-        : '';
+    let val = '' + profile.emails[0].value;
+    val.toLowerCase();
```

Comment:
We need to assign the output of `toLowerCase` to something.  What I recommend is to use `email`, which already checks that the value exists and use `email = email.toLowerCase()`.

It's probably worth checking that `typeof email === 'string'`, too, just err on the side of caution (even though it's probably a string or null).

Reply:
Hey @Sky020 looks like you used the snippet I shared as is. It was more of a pseudo code to give you an idea. I should have been more clear. 😅 

Oliver is correct here. You would need to essentially ensure that you are working with a sting and then making it lowercase before the db query.

Reply:
Right. Sorry about that. I guess I jumped the gun, and rushed too much without reviewing.